### ISSUE TITLE
Roboto 400 Wrong Rendering on Chrome

### ISSUE THREAD

**Issue by magicbruno** [google/fonts#1753]
Roboto font-weight 400 is rendered as Italic on Chrome and Opera. 

Platform Windows 10 - Italy
Chrome Versione 70.0.3538.102 (Build ufficiale) (a 64 bit)
Chrome Canary Versione 72.0.3611.0 (Build ufficiale) canary (a 32 bit)
Opera Versione:56.0.3051.99

**Comment by m4rc1e** [comment]
Have you got Roboto installed locally?

**Comment by magicbruno** [comment]
I've removed Roboto from my system, but nothing changed. I think the problem appeared about one week ago (i'm not sure) in this site: http://audis.it but I have verified same problem in other site that use Roboto 400. 
I'm sure that one month rendering was as expected. 
The problem affects only Chrome and Opera, on Firefox, Microsoft IE and Microsoft Edge, rendering work correctly.

**Comment by m4rc1e** [comment]
Add a screenshot here please.

**Comment by magicbruno** [comment]
Rendering on Firefox
![image](https://user-images.githubusercontent.com/249512/48629634-728f8f80-e9ba-11e8-8c86-800615d0e6f5.png)
Rendering on Chrome
![image](https://user-images.githubusercontent.com/249512/48629675-905cf480-e9ba-11e8-92d7-d8cff7a74162.png)

**Comment by vbourdeix** [comment]
I have Roboto installed on my machine and I'm experiencing the same issue just on a link, even if all my page uses Roboto and the rest of the text of the page is rendered correctly. Why would the display vary depending whether I have the font installed locally or not ? I don't want to remove it from my system as I sometimes use it for printing work. Moreover, I don't want to ask all of my users to remove the Roboto font from their system to be allowed to see the website without glitches.

You can check it on https://www.hobbees.fr, the 'En savoir +' button is rendered in italic on Chrome (I know, you will probably not be able to see this if it is visible only to users with the font installed on system, but just in case, I prefer giving that information).

**Comment by magicbruno** [comment]
I'm quite sure that the problem is having fonts installed locally! But this is a NEW bug! 
I have developed http://audis.it and no more that a month ago the site was rendered correctly on my machine. I didn't make any change and now robot 400 is rendered as italic. Sure, on my Windows 2012 server (no Roboto installed) is rendered correctly, but i can't say to my customer to uninstall Roboto to see the site as they expect. So. Do you think that the problem will be solved or do you suggest to me to change the font?

**Comment by vbourdeix** [comment]
I just discovered that I had the bug only with font-weight 400. No issue with font-weight 300 or 500.

**Comment by magicbruno** [comment]
Sure! The iussue concerns only Roboro 400. There

**Comment by magicbruno** [comment]
1. I've uninstalled Roboto from local machine 
2. I've cleared Chrome cache

**NOT ENOUGH**

I tried to change Google font loading method:
before:
@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,700');
embedded in my stylesheet

after:
`<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,700" rel="stylesheet" />`
in HTML head

Finally fonts are rendered correctly. 
**Conclusions**
1. It seems that if you have Roboto installed locally, in some case Chrome doesn't render font correctly.
2. This problem appeared in last two-three week. Before all did work.

**Comment by vbourdeix** [comment]
I tried multiple approaches to fix this issue. Here is what I found :

- using ` font-family: 'Roboto script all rev=1'`  or `font-family: 'Roboto script all rev=2'` allows to get a version without the italic display, but it changes the overall appearance of the font, breaking my display on areas where I use another font-weight than 400.
- a hacky way consisting of renaming the font and suggested in this SO worked for me : [https://stackoverflow.com/questions/40019408/force-chrome-to-use-external-font-in-css]

The code I use as a temporary fix is then :
```
@font-face {
  font-family: 'RobotoHobbees';
  font-style: normal;
  font-weight: 400;
  src: local('RobotoHobbees'), local('RobotoHobbees-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu7GxKOzY.woff2) format('woff2');
  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
}
/* You need to repeat this font face for as many font-weight you use */

html {
    font-family: 'RobotoHobbees', sans-serif;
}
```
The css code for the @font-face block was generated by fonts.google.com, I just changed the names.

I hope this will be helpful :-)

**Comment by rturner2** [comment]
Yes, I'm having this issue and I have the font installed locally.

**Comment by fridezlucas** [comment]
I'm having the same issue and I have Roboto installed locally. Is there any solution (instead of uninstall Roboto) ?

**Comment by magicbruno** [comment]
You may host google fonts on your server and rename it. See: [https://stackoverflow.com/questions/40019408/force-chrome-to-use-external-font-in-css](https://stackoverflow.com/questions/40019408/force-chrome-to-use-external-font-in-css)

**Comment by rturner2** [comment]
I just updated Chrome to the latest version and this issue fixed itself.

**Comment by Grienauer** [comment]
After an Firefox update the same Issue with Roboto 400 was solved on my side.

**Comment by laerm0** [comment]
Hi all. Is the problem solved for all of you? @magicbruno @vbourdeix ?

**Comment by estaub** [comment]
Related: https://github.com/google/fonts/issues/1568

**Comment by ckwien** [comment]
I have the same problem as magicbruno with Chrome 73.0.3683.103 and Win 10 1709.
Roboto is not local but will be reloaded. Firefox and IE are OK.

**Comment by zpokoin** [comment]
I just re-installed the Roboto regular font again locally. Fixed the problem.

**Comment by nickap** [comment]
For anyone that still experience the problem:

If you host the fonts in your server be sure to have the latest version.
[ You might find this helpfull: https://google-webfonts-helper.herokuapp.com/fonts/roboto?subsets=latin ]

Remove the option for the browser to use the fonts from the visitors' machine.
For example change your font-face declaration:
`@font-face {
 	font-family: Roboto;
 	font-style: normal;
 	font-weight: 300;
 	src: $font-300-eot;
        src: local('Roboto'), local('Roboto-300'), $font-300-eot-iefix format('embedded-opentype'), $font-300-woff2 format('woff2'), $font-300-woff format('woff'), $font-300-ttf format('truetype'), $font-300-svg format('svg');
 	font-display: swap;
 }`

To be like this:
`@font-face {
 	font-family: Roboto;
 	font-style: normal;
 	font-weight: 300;
 	src: $font-300-eot;
	src: $font-300-eot-iefix format('embedded-opentype'), $font-300-woff2 format('woff2'), $font-300-woff format('woff'), $font-300-ttf format('truetype'), $font-300-svg format('svg');
 	font-display: swap;
 }`

You can also use a custom name for your fonts like this:
`@font-face {
 	font-family: myCustomName;
 	font-style: normal;
 	font-weight: 300;
 	src: $font-300-eot;
	src: $font-300-eot-iefix format('embedded-opentype'), $font-300-woff2 format('woff2'), $font-300-woff format('woff'), $font-300-ttf format('truetype'), $font-300-svg format('svg');
 	font-display: swap;
 }`

**Comment by BenMorel** [comment]
Just noticed the problem on my website as well. Could someone from Google please comment on this issue? How did this appear? Is there anything you need to solve on your end?

**Comment by ryanrolds** [comment]
Seeing something odd. Roboto isn't rendering correctly, it's extra thin, on Chrome 76.0.3809.87 and Windows 10 (117763). I'm seeing the same rendering issue on pages as in the Chrome settings screen. I also see the issue in some AWS console UIs.

**Comment by KrishnaPG** [comment]
When I open any MS Word document that contains the Roboto fonts, all those characters look italic (incorrectly). So I believe the problem is with Windows, rather than Chrome. Not sure why this is happening with locally installed Roboto, though.

**Comment by ryanrolds** [comment]
The issue immediately went away when I disabled Roboto in Adobe CC Fonts.

**Comment by Octolus** [comment]
Experiencing same thing, it takes over all chrome css and makes all text bold

**Comment by Hapanmuffinssi** [comment]
And here we go again. Yesterday, 10th December 2019 everything okay. This morning I noticed Windows 10 has updated itself and every project where we use Roboto (regular) is rendered as Italic.

**Comment by hybridherbst** [comment]
I just had the same issue, started today.
It affected GMail, Hangouts, but also local Electron apps that are using Roboto!

Reinstalling Roboto from Google Fonts fixed the issue. Anyone got any clue about why this can happen? Seems really weird that a font goes corrupt overnight, I never experienced that issue before.
